packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/components/ExpertOptions/components/DemoModeToggle/DemoModeToggle.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import { CheckBox } from 'grommet'
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart';
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+export default function DemoModeToggle ({ demoMode, setDemoMode }) {
+  return (
+    <CheckBox
+      checked={demoMode}
```

Comment:
Adding `autoFocus` here might make the toggle a little easier to use from the keyboard. At the moment, you have to tab to the checkbox after the popup opens and it's not immediately obvious where the focus is.

Reply:
The expert options context menu will also have the toggle for gold standard classification too once it is implemented, so I'm not sure we want to auto focus by input.

Reply:
Good point. Autofocus could be set by the parent, on the first input. It's not a big deal, but would mean you can toggle demo mode by clicking the cog button, then hitting space/enter.